include/configs/ts7100.h

```diff
@@ -0,0 +1,224 @@
+		"else echo Failed to load kernel from eMMC;" \
+		"fi;\0" \
+	"nfsboot-kernel=if nfs ${loadaddr} ${nfsip}:${nfsroot}/boot/zImage;" \
+			"then run silowaitcharge;" \
+			"setenv bootargs root=/dev/nfs ip=dhcp " \
+			  "nfsroot=${nfsip}:${nfsroot}${nfsroot_options} rootwait rw " \
+			  "cpu_opts=0x${opts} io_opts=0x${io_opts} " \
+			  "io_model=0x${io_model} ${cmdline_append};" \
+			"bootz ${loadaddr} - ${fdtaddr};" \
+		"else echo Failed to load kernel from NFS;" \
+		"fi\0"                                             \
+	"nfsboot=echo Booting from NFS ...;"				\
+		"dhcp &&"						\
+		"if nfs ${fdtaddr} ${nfsip}:${nfsroot}/boot/boot.scr.uimg;"\
+		"then "							\
+        		"echo Booting from custom /boot/boot.scr.uimg;"	\
+			"source ${loadaddr};"				\
+		"elif nfs ${fdtaddr} ${nfsip}:${nfsroot}/boot/boot.scr;"\
+		"then "							\
+        		"echo Booting from custom /boot/boot.scr;"	\
+			"source ${loadaddr};"				\
+		"fi;"							\
+		"if nfs ${fdtaddr} ${nfsip}:${nfsroot}/boot/imx6ul-ts${model}-${io_model}.dtb;" \
+                "then "							\
+			"run nfsboot-kernel ;"                          \
+        	"fi\0"	                                		\
+  	"bootcmd_mfg=echo Booted over USB, running test/prime;" \
+		"if post;" \
+			"then ums mmc 0.1;" \
+			"mmc bootbus 0 1 0 2;" \
+			"mmc partconf 0 1 1 1;" \
+			"fuse prog -y 0 5 A070;" \
+			"fuse prog -y 0 6 10;" \
+			"fuse prog -y 0 3 300000;" \
+			"i2c mw 38 0.0 83;" \
+			"i2c mw 38 0.0 3;" \
+			"while true;" \
+				"do led green on;" \
+				"i2c mw 38 0.0 23;" \
+				"sleep 1;" \
+				"led green off;" \
+				"i2c mw 38 0.0 3;" \
+				"sleep 1;" \
+			"done;" \
+		"else echo Test Failed;" \
+			"i2c mw 38 0.0 83;" \
+			"i2c mw 38 0.0 3;" \
+			"while true;" \
+				"do led red on;" \
+				"i2c mw 38 0.0 13;" \
+				"sleep 1;" \
+				"led red off;" \
+				"i2c mw 38 0.0 3;" \
+				"sleep 1;" \
+			"done;" \
+		"fi;\0" \
+        BOOTENV
+
+#define UPDATER_ENV \
+	"update-spl=dhcp;"\
+		"if nfs ${loadaddr} ${nfsip}:${nfsroot}/boot/SPL; then " \
+                        "setexpr filesize ${filesize} / 200 ; " \
+                        "setexpr filesize ${filesize} + 1 ; " \
+                        "mmc dev 0 1 ; " \
+                        "mmc write ${loadaddr} 2 ${filesize} ; "\
+                "fi;\0" \
+        "u-boot-bin=u-boot-dtb.img\0" \
+        "u-boot-dir=boot/\0" \
+	"u-boot-flash=setexpr filesize ${filesize} / 200 ; " \
+                "setexpr filesize ${filesize} + 1 ; " \
+                "mmc dev 0 1 ; " \
+                "mmc write ${loadaddr} 8a ${filesize} ;\0" \
+	"update-uboot=dhcp;"\
+		"if nfs ${loadaddr} ${nfsip}:${nfsroot}/${u-boot-dir}${u-boot-bin}; then " \
+                        "setexpr filesize ${filesize} / 200 ; " \
+                        "setexpr filesize ${filesize} + 1 ; " \
+                        "mmc dev 0 1;" \
+                        "mmc write ${loadaddr} 8a ${filesize};"\
+                "fi;\0"
```

Comment:
```suggestion
```

This appears unused and should not be included.  This might be better as separate internal documentation.

Reply:
Removed.